en_ewt-ud-train.conllu

```diff
@@ -230713,7 +230713,7 @@
 2	2010	2010	NUM	CD	NumType=Card	3	nummod	3:nummod	_
 3	Genesis	Genesis	PROPN	NNP	Number=Sing	5	nsubj	5:nsubj	_
 4	is	be	AUX	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	5	cop	5:cop	_
-5	grrrrrrrreeeaaat	great	ADJ	JJ	Degree=Pos	0	root	0:root	SpaceAfter=No
+5	grrrrrrrreeeaaat	great	ADJ	JJ	Degree=Pos|Typo=Yes	0	root	0:root	SpaceAfter=No|CorrectForm=great
```

Comment:
Stylistic lengthening

Reply:
`CorrectForm=great`

Reply:
I think that CorrectForm should be limited to tokens with Typo=Yes. The real correct form is what the author wrote, the lemma indicates the "base form" and in fact the CorrectForm will be most often be the same as the lemma.

Reply:
The reason to have a CorrectForm is that the word might have an inflectional ending that gets lost in the lemma.